docs/docs/community/maintainers/saved-replies.md

```diff
@@ -123,17 +123,19 @@ It may be helpful to store some of these in [GitHub’s saved replies](https://g
 
 ### Code of conduct
 
-<!-- TODO: Add a link to CoC enforcement committee members list. -->
 :::note
-The following replies should only be used by Nebari's Code of Conduct committee members. 
+The following replies should only be used by [Nebari's Code of Conduct committee members][nebari-coc-committee].
 :::
 
 - **First warning**
-<!-- TODO: add link to CoC -->
+
   ```md
-  [[at-mention]] your comment was [[minimized|deleted]] as a violation of the Nebari Code of Conduct. You may consider this an official warning.
+  [[at-mention]] your comment was [[minimized|deleted]] as a violation of the [Nebari Code of Conduct](https://github.com/nebari-dev/governance/blob/main/CODE_OF_CONDUCT.md). You may consider this an official warning.
```

Comment:
Note, this is an explicit link because it's something folks can "copy" directly. :)